core/invoker/src/main/resources/application.conf

```diff
@@ -87,6 +86,10 @@ whisk {
     #  2. OR yaml formatted multi line string. See multi line config support https://github.com/lightbend/config/blob/master/HOCON.md#multi-line-strings
     #
     #pod-template =
+
+    # Set this optiona string to be the namespace that the invoker should target for adding pods. This allows the invoker to run in a namesapce it doesn't have API access to but add pods to another namespace. See also https://github.com/apache/openwhisk/issues/4711
```

Comment:
Nit `optiona` -> `optional`

Also explain the default behaviour when this is not configured i.e. actions pod would be launched in current invoker pods namespace